README.md

```diff
@@ -1,26 +1,163 @@
+
+| Type | Name | Language | Note |
+|------|------|----------|------|
+| Reference | [bc-lifehash](https://github.com/BlockchainCommons/bc-lifehash) | C++/C
+| Reference | [LifeHash](https://github.com/BlockchainCommons/LifeHash) | Swift
+| Reference | [LifeHash](https://github.com/BlockchainCommons/LifeHash) | Mathematica/Wolfram Language | `version1` only
+
+## Command-Line Tools
+
+Command-line reference tools demonstrate our  libraries by generating LifeHash images as PNG files.
 
-## Installation
+| Name | Language |
+|-----|-----|
+| [lifehash-cli](https://github.com/BlockchainCommons/bc-lifehash-cli) | C++ |
+| [LifeHashTool](https://github.com/BlockchainCommons/LifeHashTool) | Swift |
 
-Prerequsites: Node, Emscripten
+## Gordian Principles
+
+**LifeHash** is a specification meant to display the [Gordian Principles](https://github.com/BlockchainCommons/Gordian#gordian-principles), which are philosophical and technical underpinnings to Blockchain Commons' Gordian technology. This includes:
+
+* **Independence.** LifeHash allows you to personally verify data, rather than depending entirely on applications.
+* **Privacy.** LifeHash improves your privacy by allowing identification through [functional identity](https://github.com/WebOfTrustInfo/rwot9-prague/blob/master/topics-and-advance-readings/functional-identity-primer.md).
+* **Resilience.** LifeHash helps to protect from scams built on unverified fake data.
+* **Openness.** LifeHash is an open specification.
+
+## LifeHash.info Demo Installation Instructions
+
+To install your own copy of the [LifeHash.info website](https://lifehash.info):
+
+* **Prerequsites:** Node, Emscripten
 
 ```
-git clone https://github.com/BlockchainCommons/lifehash-web/
-cd lifehash-web
+git clone https://github.com/BlockchainCommons/lifehash.info/
+cd lifehash.info
 git submodule update --init
 npm install
 ./build_wasm.sh
 npm run dev
 ```
+
+## LifeHash.Info Demo Usage Instructions
+
+The [LifeHash.info website](https://lifehash.info) allows you to view LifeHashes for UTF-8 strings and SHA-256 hashes, which you can enter on the page itself, or request through a URL.
+
+### Requesting a Specific LifeHash
+
+You can use a `s=` query parameter in the URL to populate the string input field with a specific value. For example:
+
+[http://lifehash.info?s=RBH-AOZ](http://lifehash.info?s=RBH-AOZ)
+
+If you want to generate a LifeHash directly from a SHA-256 digest, you can use a `f=` query parameter in the URL:
+
+[http://lifehash.info?f=70b0db749f1cb026282693b236048cefa3395fc580e91b7c62666b0bdc97c0e3](http://lifehash.info?f=70b0db749f1cb026282693b236048cefa3395fc580e91b7c62666b0bdc97c0e3)
```

Comment:
This should be `d=`, not `f=`.

Reply:
I'd wondered what the f= stood for! Fixed.